src/web/routes/application/in-scotland/index.js

```diff
@@ -0,0 +1,5 @@
+const { iLiveInScotland } = require('./in-scotland')
```

Comment:
Needs updating to match updated export. Could do the following to avoid future updates:
```
module.exports = require('./in-scotland')
```